manifests/config.pp

```diff
@@ -29,10 +29,16 @@
   $version = $::nexus::version,
 ) {
 
-  if $version !~ /\d.*/ or versioncmp($version, '3.0.0') >= 0 {
-    $conf_path = 'etc/org.sonatype.nexus.cfg'
-  } else {
-    $conf_path = 'conf/nexus.properties'
+  case $version {
```

Comment:
because when the version is above 3.0.0, for example 3.1.0, the function returns 1 and the wrong config file path set.
version 3.1.0 uses 'etc/nexus.properties', but as I've found out though, it should be placed in the nexus data directory.
I'm doing a rewrite of the module addressing this issue and some others because I need a working 3.1.0 nexus server this week.